src/test/scala/chiselTests/util/SizeOfSpec.scala

```diff
@@ -0,0 +1,39 @@
+package chiselTests.util
+
+import chisel3._
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage}
+import chisel3.testers.BasicTester
+import chisel3.util.SizeOf
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.io.Source
+
+class SizeOfTop extends Module {
+  val io = IO(new Bundle{
+    val w = Input(UInt(65.W))
+    val outw = UInt(32.W)
+  })
+  io.outw  := SizeOf(io.w)
+}
+
+/** A test for intrinsics.  Since chisel is producing intrinsics as tagged 
+ * extmodules (for now), we explicitly test the chirrtl and annotations rather
+ * than the processed firrtl or verilog.  It is worth noting that annotations 
+ * are implemented (for now) in a way which makes the output valid for all
+ * firrtl compilers, hence we write a localized, not end-to-end test */
+class SizeOfSpec extends AnyFlatSpec with Matchers {
```

Comment:
my comment above this could just be IntrinsicSpec and we define the Sizeof intrinsic here... I'm unclear why anyone would want the sizeof functionality in hardware design